node/cmd/guardiand/node.go

```diff
@@ -924,12 +924,14 @@ func runNode(cmd *cobra.Command, args []string) {
 		//
 		// NOTE:  The "none" is a special indicator to disable a watcher until it is desirable to turn it back on.
 
+		var ethWatcher *evm.Watcher
 		if shouldStart(ethRPC) {
 			logger.Info("Starting Ethereum watcher")
 			readiness.RegisterComponent(common.ReadinessEthSyncing)
 			chainObsvReqC[vaa.ChainIDEthereum] = make(chan *gossipv1.ObservationRequest, observationRequestBufferSize)
+			ethWatcher = evm.NewEthWatcher(*ethRPC, ethContractAddr, "eth", common.ReadinessEthSyncing, vaa.ChainIDEthereum, lockC, setC, 1, chainObsvReqC[vaa.ChainIDEthereum], false, nil)
```

Comment:
Tests are failing because `*unsafeDevMode` isn't getting passed anymore and it's trying to call for finalized blocks. I think @kev1n-peters is trying to update ganache to support this!

Reply:
Oops! Fixed.